src/Workspaces/Core/Portable/Remote/RemoteArguments.cs

```diff
@@ -118,14 +42,14 @@ public bool Equals(SerializableSymbolAndProjectId other)
             return new SerializableSymbolAndProjectId
             {
                 SymbolKeyData = symbolAndProjectId.Symbol.GetSymbolKey().ToString(),
-                ProjectId = SerializableProjectId.Dehydrate(symbolAndProjectId.ProjectId)
+                ProjectId = symbolAndProjectId.ProjectId
```

Comment:
Does ths work when the serializable type is used as aa property of another type?